x/superfluid/keeper/unpool_test.go

```diff
@@ -252,3 +251,37 @@ func (suite *KeeperTestSuite) TestUnpool() {
 		})
 	}
 }
+
+// TestUnpoolAllowedPools_WhiteList tests that unpooling does not work for pools not in a whitelist.
+// Should fail immediately if pool is not in whitelist.
+func (suite *KeeperTestSuite) TestUnpoolAllowedPools_WhiteList() {
+	// lock id does not matter in the context of this test.
+	const (
+		testLockId        = 1
```

Comment:
whats the point of this const? I don't see it used elsewhere

Reply:
fixed